spec/requests/cloud_subnets_spec.rb

```diff
@@ -77,4 +77,17 @@
       expect(response).to have_http_status(:forbidden)
     end
   end
+
+  describe 'security groups subcollection' do
+    it "can list a subnet's security groups" do
+      subnet = FactoryGirl.create(:cloud_subnet)
+      subnet.security_groups = [FactoryGirl.create(:security_group)]
+      api_basic_authorize(action_identifier(:cloud_subnets, :read, :subcollection_actions, :get))
```

Comment:
@abellotti or @agrare maybe: is there some documentation available about config/api.yaml? I'm getting 

```
{"error" => {"kind" => "forbidden", "message" => "Use of the read action is forbidden" ...
```

Error here no matter what. Funny thing is we have exactly same test and configuration on the other PR and it works there... I'm not sure what am I doing wrong this time.

Reply:
some write-up there in https://github.com/ManageIQ/manageiq-api/blob/master/CONTRIBUTING.md